pkg/sdk/accounts.go

```diff
@@ -0,0 +1,375 @@
+	StatementTimeoutInSeconds       *bool `ddl:"keyword" db:"STATEMENT_TIMEOUT_IN_SECONDS"`
+
+	// Session params
+	AbortDetachedQuery               *bool `ddl:"keyword" db:"ABORT_DETACHED_QUERY"`
+	Autocommit                       *bool `ddl:"keyword" db:"AUTOCOMMIT"`
+	BinaryInputFormat                *bool `ddl:"keyword" db:"BINARY_INPUT_FORMAT"`
+	BinaryOutputFormat               *bool `ddl:"keyword" db:"BINARY_OUTPUT_FORMAT"`
+	DateInputFormat                  *bool `ddl:"keyword" db:"DATE_INPUT_FORMAT"`
+	DateOutputFormat                 *bool `ddl:"keyword" db:"DATE_OUTPUT_FORMAT"`
+	ErrorOnNondeterministicMerge     *bool `ddl:"keyword" db:"ERROR_ON_NONDETERMINISTIC_MERGE"`
+	ErrorOnNondeterministicUpdate    *bool `ddl:"keyword" db:"ERROR_ON_NONDETERMINISTIC_UPDATE"`
+	JsonIndent                       *bool `ddl:"keyword" db:"JSON_INDENT"`
+	LockTimeout                      *bool `ddl:"keyword" db:"LOCK_TIMEOUT"`
+	QueryTag                         *bool `ddl:"keyword" db:"QUERY_TAG"`
+	RowsPerResultset                 *bool `ddl:"keyword" db:"ROWS_PER_RESULTSET"`
+	SimulatedDataSharingConsumer     *bool `ddl:"keyword" db:"SIMULATED_DATA_SHARING_CONSUMER"`
+	StrictJsonOutput                 *bool `ddl:"keyword" db:"STRICT_JSON_OUTPUT"`
+	TimestampDayIsAlways24h          *bool `ddl:"keyword" db:"TIMESTAMP_DAY_IS_ALWAYS_24H"`
+	TimestampInputFormat             *bool `ddl:"keyword" db:"TIMESTAMP_INPUT_FORMAT"`
+	TimestampLtzOutputFormat         *bool `ddl:"keyword" db:"TIMESTAMP_LTZ_OUTPUT_FORMAT"`
+	TimestampNtzOutputFormat         *bool `ddl:"keyword" db:"TIMESTAMP_NTZ_OUTPUT_FORMAT"`
+	TimestampOutputFormat            *bool `ddl:"keyword" db:"TIMESTAMP_OUTPUT_FORMAT"`
+	TimestampTypeMapping             *bool `ddl:"keyword" db:"TIMESTAMP_TYPE_MAPPING"`
+	TimestampTzOutputFormat          *bool `ddl:"keyword" db:"TIMESTAMP_TZ_OUTPUT_FORMAT"`
+	Timezone                         *bool `ddl:"keyword" db:"TIMEZONE"`
+	TimeInputFormat                  *bool `ddl:"keyword" db:"TIME_INPUT_FORMAT"`
+	TimeOutputFormat                 *bool `ddl:"keyword" db:"TIME_OUTPUT_FORMAT"`
+	TransactionDefaultIsolationLevel *bool `ddl:"keyword" db:"TRANSACTION_DEFAULT_ISOLATION_LEVEL"`
+	TwoDigitCenturyStart             *bool `ddl:"keyword" db:"TWO_DIGIT_CENTURY_START"`
+	UnsupportedDdlAction             *bool `ddl:"keyword" db:"UNSUPPORTED_DDL_ACTION"`
+	UseCachedResult                  *bool `ddl:"keyword" db:"USE_CACHED_RESULT"`
+	WeekOfYearPolicy                 *bool `ddl:"keyword" db:"WEEK_OF_YEAR_POLICY"`
+	WeekStart                        *bool `ddl:"keyword" db:"WEEK_START"`
+}
+
+func (c *accounts) Alter(ctx context.Context, opts *AccountAlterOptions) error {
+	if opts == nil {
+		opts = &AccountAlterOptions{}
+	}
+	if err := opts.validate(); err != nil {
+		return err
+	}
+	sql, err := structToSQL(opts)
+	if err != nil {
+		return err
+	}
+	_, err = c.client.exec(ctx, sql)
+	return err
+}
+
+type AccountShowOptions struct {
```

Comment:
Since there is also a "SHOW MANAGED ACCOUNTS" command it would make more sense to called this ShowOrganizationAccounts or something like that